### PR TITLE
DO NOT MERGE!!! This PR is meant to test what in package-lock.json breaks netlify

### PR DESCRIPTION
**Pull request by sielay**

### CODE REVIEW

**Comment by sielay** [comment]
By luck found that. It refers to same problems. That has to do something with webpack config, so ned to dig into gatsby again.

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "start": "gatsby develop",
     "develop": "gatsby develop",
-    "build": "gatsby build",
+    "build": "npm run patch && gatsby build",
```
Comment (sielay) [inline package.json]: https://github.com/gatsbyjs/gatsby/issues/1846

File: package.json [hunk 2]
```diff
@@ -40,11 +41,11 @@
     "gatsby-plugin-google-analytics": "latest",
     "gatsby-plugin-manifest": "latest",
     "gatsby-plugin-offline": "latest",
-    "gatsby-plugin-sharp": "latest",
+    "gatsby-plugin-sharp": "^1.6.31",
```
Comment (sielay) [inline package.json]: https://github.com/gatsbyjs/gatsby/issues/4131

File: src/layouts/index.tsx [hunk 3]
```diff
@@ -15,18 +17,18 @@ export const menuItems = [
...
-  render() {
-    const { pathname } = this.props.location;
-    const isHome = pathname === "/";
+export default (props: DefaultLayoutProps): JSX.Element => {
+  const { pathname } = props.location;
+  const isHome = pathname === "/";
 
-    return (
+  return (
+    <Provider store={store}>
```
Comment (sielay) [inline src/layouts/index.tsx]: https://github.com/fabien0102/gatsby-starter/issues/54

File: src/layouts/index.tsx [hunk 4]
```diff
@@ -15,18 +17,18 @@ export const menuItems = [
...
   };
-  children: any;
+  children?: any;
 }
 
-export default class DefaultLayout extends React.PureComponent<DefaultLayoutProps, void> {
-  render() {
-    const { pathname } = this.props.location;
-    const isHome = pathname === "/";
+export default (props: DefaultLayoutProps): JSX.Element => {
```
Comment (sielay) [inline src/layouts/index.tsx]: Some changes around typings make that TS is picky about class form components here.

**Comment by fabien0102** [comment]
@sielay I close this PR, normally it's resolve 😄

**Comment by sielay** [comment]
Is it? Ok. Let's do it. I think we still have a problem when upgrading packages. Will take a look.

**Comment by fabien0102** [comment]
If you have any needs, you can of course reopen this PR :wink: It's just to cleanup a little bit my "todo list" ^^

**Pull request closed without merging**